_posts/2022-mm-dd-crypto-1/2022-mm-dd-crypto1-history-and-basics.markdown

```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title:  "Cryptography 1 - history and basics"
+date:   2022-mm-dd HH:MM:SS +0100
+categories: cryptography
+---
+# Cryptography
+## Part I: history and basics
+Hi everyone, Fnord here.
+This post is the first in a small series on cryptography, written in collaboration with [CRTified](https://github.com/CRTified). For the first post, we'll mainly focus on the history of cryptography and set some basic ground rules. We'll also talk about what "crypto systems" are and take a look at a simple method of encrypting texts.
+
+## Table of Contents
+* History
+* Notation/Ground rules
+* What are crypto systems?
+  * Symmetric
+    * Caesar Cipher - a simple symmetric encryption
+  * Asymmetric
+
+### History
+_Alt title: "what is cryptography anyways?"_
+Don't worry, I'll spare you the `n + 1`[st|nd|th] detailed reiteration of things you can find on [Wikipedia](https://en.wikipedia.org/wiki/Cryptography), but let's at least recap a few major points:
+* For as long as recorded history goes, people have felt the need to hide crucial or personal information from others. Earliest examples of what could be considered cryptography date back to Ancient Greece, India and Persia.
+* Most classical crypto algorithms relied on either swapping the order of letters or the letters themselves, up until the 15th century when so-called polyalphabetic ciphers introduced more than one possible substitution per letter. We'll talk about those in another post.
+* In the 19th century, specifically in 1883 with Auguste Kerkhoffs stating his famous principle "the enemy knows the system", the crypto world finally recognized that security _must_ come from the encryption key, not the encryption method.
+* Computers kick-started modern cryptography, because suddenly it became very easy to perform complex operations very quickly. They also moved cryptography from being something that mostly military personnel and intelligence agencies were concerned with into the academic and public domain.
+* The age of modern computing, with its ever-increasing amount of computational power (quantum computing, anyone?) poses new problems and threats to proven approaches.
+
+"But where does he want to go with this?", you may want to ask. At this point, cryptography is something that concerns us all. It's in our phones, our banks, our computers and even in our refrigerators and ceiling lamps (hopefully). And I think it's important to understand what's going on. To a degree at least. And to understand where we stand and where we're going, we need to understand where we come from. If you take anything from this paragraph, let it be "the enemy knows the system" - never trust complexity as a security device.
+
+To answer the initial question posed in the alt title: cryptography is a means of ensuring that important information can only be accessed by people/parties you're comfortable sharing it with.
```

Comment:
But cryptography is much more (Signatures/MACs do not provide confidentiality, they have "other goals").

I'd love to add a short section about Crypto**logy** in general, which can be seen as the field of Cryptography ("Defending primitives"), Cryptanalysis ("Attacking primitives") and maybe Protocols ("Combining Primitives").

Otherwise, it might get hart to justify the need for e.g. signatures or hash functions.

Reply:
Agreed - I just kinda felt like I'd already begun rambling in that part, so I wanted to wrap it up. But I share your thoughts, broadening the picture here might be better in the context of future content.